lib/fpm/package/deb.rb

```diff
@@ -267,6 +267,22 @@ def extract_info(package)
       #self.config_files = config_files
 
       self.dependencies += parse_depends(parse.call("Depends")) if !attributes[:no_auto_depends?]
+
+      if File.file?(File.join(path, "preinst"))
+        self.scripts[:before_install] = File.read(File.join(path, "preinst"))
+      end
+      if File.file?(File.join(path, "postinst"))
+        self.scripts[:after_install] = File.read(File.join(path, "postinst"))
+      end
+      if File.file?(File.join(path, "prerm"))
+        self.scripts[:before_remove] = File.read(File.join(path, "prerm"))
+      end
+      if File.file?(File.join(path, "postrm"))
+        self.scripts[:after_remove] = File.read(File.join(path, "postrm"))
+      end
+      if File.file?(File.join(path, "conffiles"))
+        self.config_files = File.read(File.join(path, "conffiles")).split("\n")
```

Comment:
Forgot to mention this -- the pull request also preserves the list of config files when converting from debian packages. Just tested this (just now!) :+1: Works when translating to an RPM package, whether one config file or more than one is originally listed in the debian package.